lib/spree/backend/engine.rb

```diff
@@ -37,12 +37,15 @@ class Engine < ::Rails::Engine
         Rails.application.config.spree_backend.actions[:users] = Spree::Admin::Actions::UsersDefaultActionsBuilder.new.build
         Rails.application.config.spree_backend.actions[:user] = Spree::Admin::Actions::UserDefaultActionsBuilder.new.build
         Rails.application.config.spree_backend.actions[:products] = Spree::Admin::Actions::ProductsDefaultActionsBuilder.new.build
-        Rails.application.config.spree_backend.actions.include?(:images) ? (Rails.application.config.spree_backend.actions[:images].items << Spree::Admin::Actions::ImagesDefaultActionsBuilder.new.build.items).flatten! : Rails.application.config.spree_backend.actions[:images] = Spree::Admin::Actions::ImagesDefaultActionsBuilder.new.build
+        Rails.application.config.spree_backend.actions[:product] = Spree::Admin::Actions::Root.new
```

Comment:
Do we have any guarantee that the config changes from `spree_backend` will be executed before the config changes from `spree_rails_frontend`? 🤔
If yes - then this is ok.
If not - then there's a risk tat `Spree::Admin::Actions::Root.new` would override the contents that were assigned during the `after_initialize` in the `spree_rails_frontend`.

Presumably this can be avoided by having the gems being listed in the `Gemfile` in a specific order but we have no control over how people compose their Gemfiles.

Reply:
We can add a check in spree_rails_frontend, that ensures that spree_backend is initialized first, and if not, throws an exception that tells users what to do :)

Reply:
Another option I was exploring was to run the initialization process in a regular initializer, but due to how preferences are built right now, in order to load routes Spree needs to establish a database connection. I think if we solve this, we could make it independent on the order - this is however something to be fixed on its own separately, unfortunately this looks like a bigger rewrite.